project/CelebornBuild.scala

```diff
@@ -553,7 +573,7 @@ trait SparkClientProjects {
         ) ++ commonUnitTestDependencies
       )
   }
-  
```

Comment:
should it be covered by spotless?

Reply:
after experimentation, it seems that the maven spotless plugin is unable to format source files located outside the source directory.

Reply:
not a big deal.